<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumPopup</class>
 <widget class="QDialog" name="AlbumPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Album</string>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>430</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="cover_art">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Default_Images/default_track.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ALBUM_TITLE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="artist_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ARTIST_NAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="total_duration">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 hr 20 minutes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NOTE">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TRACKS ORDERED BY TRACKLIST_NUM, WITH GAPS INBETWEEN IF MISSING!!!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>EDIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date (long form)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tracks_table">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>551</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="type">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <zorder>button_box</zorder>
  <zorder>cover_art</zorder>
  <zorder>title</zorder>
  <zorder>artist_name</zorder>
  <zorder>total_duration</zorder>
  <zorder>edit_button</zorder>
  <zorder>date</zorder>
  <zorder>tracks_table</zorder>
  <zorder>NOTE</zorder>
  <zorder>type</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>AlbumPopup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>AlbumPopup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
